incubator/buzzfeed-sso/templates/proxy-deployment.yaml

```diff
@@ -61,8 +61,18 @@ spec:
                 secretKeyRef:
                   name: {{ $proxySecret }}
                   key: proxy-cookie-secret
+            {{- if .Values.proxy.defaultAllowedEmailDomains }}
+            {{- if (eq "-" .Values.proxy.defaultAllowedEmailDomains) }}
```

Comment:
I considered that, i think empty string and a non present value will be considered an "invalid" type? So i cant then fall back to the emailDomain and make it backwards compatible? What do you think?

Reply:
so we can't set it to an empty string because then `{{- if .Values.proxy.defaultAllowedEmailDomains }}` would be evaluated to `false` (that's why we force the `-` pattern)

I think this is the best pattern we can go with, the sprig way is with a third party